data/LogicHelpers.json

```diff
@@ -20,7 +20,7 @@
     # "Bow": "'Bow'",
     # "Slingshot": "'Slingshot'",
     "Bombs": "Bomb_Bag",
-    "Deku_Shield": "Buy_Deku_Shield",
+    "Deku_Shield": "Buy_Deku_Shield or Deku_Shield_Drop",
```

Comment:
Are we sure that this overrides the normal behavior wherein the item "Deku Shield" (i.e. a normal non repeatable one) would be used where `Deku_Shield` is referenced?

Reply:
That seems like the sort of thing that would have been noticed by now if it was broken.

Reply:
Well before now it was using `Buy_Deku_Shield` directly

Reply:
Ah, I didn't realize that. Still, the `Goron_Tunic` helper is in use, and I'm positive that one works as intended.

Reply:
Good point, I think we can probably call this safe then.